Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -664,7 +703,7 @@ def _stringify_lists_if_needed(event):
     shared_with = event.get("sharedWith")
     private_ip_addresses = event.get("privateIpAddresses")
     if shared_with:
-        shared_list = [u["cloudUsername"] for u in shared_with]
+        shared_list = [u.get("cloudUsername") for u in shared_with]
```

Comment:
I'm assuming you want an empty list here if `"cloudUsername"` isn't in any of `shared_with`'s items. This will currently return a list of `None`s:

```
>>> shared_with = [{"key": "value"}, {"key": "value2"}]
>>> [u.get("cloudUsername") for u in shared_with]
[None, None]
```
You'll want:
```
>>>[u.get("cloudUsername") for u in shared_with if u.get("cloudUsername")]
[]
```